src/components/App.tsx

```diff
@@ -252,8 +253,12 @@ import {
   isLocalLink,
 } from "../element/Hyperlink";
 
-const IsMobileContext = React.createContext(false);
-export const useIsMobile = () => useContext(IsMobileContext);
+const defaultDeviceInfoContext: DeviceInfo = {
+  isMobile: false,
+  isTouchScreen: false,
+};
+const DeviceInfoContext = React.createContext(defaultDeviceInfoContext);
```

Comment:
rename to `DeviceTypeContext` ?

Reply:
Do you also want to rename `deviceInfo` to `deviceTypeInfo` and `useDeviceInfo()` to `useDeviceTypeInfo()`?

or `deviceInfo` to `deviceType` and `useDeviceInfo()` to `useDeviceType()`?

Reply:
`deviceInfo` to `deviceType` and `useDeviceInfo()` to `useDeviceType()` :)